### PR TITLE
tox: update to 3.28.0, orphan.

### PR DESCRIPTION
**Pull request by Chocimier**
#### Testing the changes
- I tested the changes in this PR: **briefly**

Tox was packaged for do_check, but it runs in virtualenv, thus not useful.
Maybe it could be made work with [sitepackages](https://tox.wiki/en/latest/config.html#sitepackages).

### CODE REVIEW

File: srcpkgs/tox/template [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 depends="python3-filelock python3-packaging
  python3-pluggy python3-pytest python3-toml python3-virtualenv"
 checkdepends="${depends} python3-pip python3-pytest-mock
  python3-flaky python3-pathlib2 python3-freezegun"
 short_desc="Generic virtualenv management and test command line tool"
-maintainer="Piotr Wójcik <chocimier@tlen.pl>"
+maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://tox.readthedocs.io/en/latest/"
 changelog="https://tox.readthedocs.io/en/latest/changelog.html"
```
Comment (classabbyamp) [inline srcpkgs/tox/template:15]:
```suggestion
homepage="https://tox.wiki/en/legacy/"
changelog="https://tox.wiki/en/legacy/changelog.html"
```
redirects, also switch branch for 3.x

File: srcpkgs/tox/template [hunk 2]
```diff
@@ -1,25 +1,26 @@
 # Template file for 'tox'
 pkgname=tox
-version=3.25.1
-revision=2
+version=3.28.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools_scm"
 depends="python3-filelock python3-packaging
  python3-pluggy python3-pytest python3-toml python3-virtualenv"
```
Comment (classabbyamp) [inline srcpkgs/tox/template:8]:
```suggestion
 python3-pluggy python3-pytest python3-virtualenv"
```
uses builtin tomllib now

File: srcpkgs/python3-pytest/template [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'python3-pytest'
 pkgname=python3-pytest
-version=7.1.3
+version=7.2.1
 revision=1
 build_style=python3-module
 _common_deps="python3-attrs python3-iniconfig python3-py python3-pluggy"
```
Comment (classabbyamp) [inline srcpkgs/python3-pytest/template:6]: remove python3-tomli below, now uses builtin tomllib

File: srcpkgs/tox/template (hunk 1 above)
Comment (Chocimier) [reply srcpkgs/tox/template]: OK, even if ignoring 4.* wasn't intended to commit.

**Pull request merged**